doc/configuration.md

```diff
@@ -25,3 +25,4 @@ The following settings are provided:
   - `.` - Unsuccessful shrink
   - `!` - Successful shrink
 - `reproduce` - Opaque string that encodes the information necessary to reproduce minimal failures for properties. Since this string is opaque, it can only be obtained from a failed RapidCheck run. Refer to the [debugging documentation](debugging.md) for more information.
+- `shrink_tries` - Number of times each shrink is tested. Defaults to `1`.
```

Comment:
I think you want to be able to adjust this manually. It would be good to ignore it in the sequential case however, since it won't do any good there...